infra/nnfw/python/nnfwapi/__init__.py

```diff
@@ -0,0 +1,2 @@
+__all__ = ['libnnfw_api_pybind']
+from . import libnnfw_api_pybind
```

Comment:
Why `infra` needs to have this python script?

Reply:
We have libnnfw_api_pybind.py because

It help user import libnnfw_api_pybind that matches the architecture automatically.
It serves as the interface between libnnfw_api_pybind.so and the user. (nnfw_session, tensorinfo)
We used this approach of placing different SO files with the same name in architecture-specific folders to distinguish them.
Is there an alternative approach available for achieving this?

Reply:
I am okay for the content of `*.py` themselves. I wonder why it is located under `infra`.

Reply:
> **init**.py, libnnfw_api_pybind.py need to be part of a package. How about moving **init**.py, libnnfwa_api_pybind.py in 'ONE/runtime/onert/python/'?

I don't take a look in detail. But if it is the code for typical user, it would be better be a part of python package, and put somewhere under `runtime/onert/python`, not under `infra`.

Reply:
Ok, thanks for your guidance,
We will update